scripts/wasm-all.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # Detect the architecture #
-if [[ $(arch) == "arm64" ]]; then
+if [[ $(arch) == "arm64a" ]]; then
```

Comment:
Was it typo, or now it is? :D

Reply:
whoeps, tried compiling with x86 on arm but took way too long. good spot.